service_container/factories.rst

```diff
@@ -100,6 +100,81 @@ create its object:
     the configured class name may be used by compiler passes and therefore
     should be set to a sensible value.
 
+You can also use the ``factory`` option to create services from static methods.
+Lets suppose the ``NewsletterManager`` class has a ``create()`` method that needs
```

Comment:
Let's
but I don't get your PR: the existing example (before this one) uses a static method already.